lib/pbench/server/api/resources/pbench_users.py

```diff
@@ -333,31 +447,40 @@ def delete(self, username):
         """
         # get the auth token
         auth_header = request.headers.get("Authorization")
-        auth_token = ""
-        if auth_header:
-            try:
-                auth_token = auth_header.split(" ")[1]
-            except IndexError:
-                self.app.logger.info(
-                    f"failed to delete the user due to malformed Bearer token: {auth_header}"
-                )
-                abort(401, message="Bearer token malformed, failed to delete the user")
 
-        if auth_token:
+        if not auth_header:
+            self.app.logger.error(f"Auth token not provided while deleting a user")
+            abort(
+                400,
+                message="Please add 'Authorization' token as Authorization: Bearer <JWT_Auth_token>",
+            )
+
+        try:
+            auth_token = auth_header.split(" ")[1]
```

Comment:
And let's do this just once in a common method. I suspect it'd make sense to validate it, too.